.env.example

```diff
@@ -1,26 +1,37 @@
 # terraform variables
-TF_VAR_AWS_PROFILE=default# do not change this
-TF_VAR_VM_NAME=YOUR_IDENTIFIER# default "polygon-user". It can be any string, used to discriminate between instances
-TF_VAR_DOCKERIZED=no# default "no", otherwise only one VM is created and the Polygon devnet will run in docker containers
-TF_VAR_DISK_SIZE_GB=100# size of the disk in GB (default is 100GB)
-TF_VAR_ARCHIVE_DISK_SIZE_GB=100 # size of the disk in GB in archive node (default is 100GB)
-TF_VAR_IOPS=3000# Amount of provisioned IOPS
-TF_VAR_ARCHIVE_IOPS=15000 # Amount of provisioned IOPS in archive node
-TF_VAR_VALIDATOR_COUNT=2# number of validator nodes (default is 2). Note that while spinning up a public network (mainnet/mumbai) node, this will serve as a non-validator
-TF_VAR_SENTRY_COUNT=1# number of non-validator sentry nodes (default is 1)
-TF_VAR_ARCHIVE_COUNT=0 # number of archive nodes (default is 0)
-TF_VAR_INSTANCE_TYPE=t2.xlarge# type of the EC2 VM instance (default is t2.xlarge)
-TF_VAR_ARCHIVE_INSTANCE_TYPE=t2.xlarge # default t2.xlarge
-TF_VAR_VOLUME_TYPE=gp3 # default gp3
-TF_VAR_ARCHIVE_VOLUME_TYPE=io1 # type of EBS volume for archive nodes (default is io1)
-TF_VAR_INSTANCE_AMI=ami-017fecd1353bcc96e# ami instance type (default is ami-017fecd1353bcc96e, running ubuntu 22.04)
-TF_VAR_PEM_FILE=aws-key# name of the certificate (.pem file) previously generated (default is aws-key)
-TF_VAR_REGION=us-west-2# AWS region to deploy to (default is us-west-2)
-PEM_FILE_PATH=/absolute/path/to/your/aws-key.pem# absolute path pointing to the certificate previously downloaded
+TF_VAR_AWS_PROFILE=default # do not change this
+TF_VAR_VM_NAME=YOUR_IDENTIFIER # default "polygon-user". It can be any string, used to discriminate between instances
+TF_VAR_DOCKERIZED=no # default "no", otherwise only one VM is created and the Polygon devnet will run in docker containers
+TF_VAR_BOR_DISK_SIZE_GB=20 # size of the disk in GB for Bor client (default is 100GB)
+TF_VAR_ERIGON_DISK_SIZE_GB=20 # size of the disk in GB for Erigon client (default is 100GB)
+TF_VAR_BOR_ARCHIVE_DISK_SIZE_GB=100 # size of the disk in GB in Bor archive node (default is 100GB)
+TF_VAR_ERIGON_ARCHIVE_DISK_SIZE_GB=100 # size of the disk in GB in Erigon archive node (default is 100GB)
+TF_VAR_BOR_IOPS=3000 # Amount of provisioned IOPS for Bor client
+TF_VAR_ERIGON_IOPS=3000 # Amount of provisioned IOPS for Erigon client
+TF_VAR_BOR_ARCHIVE_IOPS=15000 # Amount of provisioned IOPS in Bor archive node
```

Comment:
I was just trying with lower sized disk and we cannot have high IOPS with such small disks 😅

Reply:
I'll have to check for the limit, 1000 worked for me but I'm not sure about the max/prescribed limit